rotkehlchen/tests/utils/globaldb.py

```diff
@@ -19,7 +19,8 @@
         name='Custom 1',
         symbol='CST1',
         started=Timestamp(0),
-        swapped_for=A_MKR,
+        # Very ugly (swapped_for is CryptoAsset) but this constant is used for parametrizing tests.
+        swapped_for=Asset(A_MKR.identifier),  # type: ignore
```

Comment:
wait whaaaat?

Reply:
How was this in this review? Is a change that alexei did. The type ignore is there before the swapped_for is expected to be a cryptoasset but that would require fully initializing the asset so he added a type ignore. As for why he did `Asset(MKR.identifier)` I believe he made a replace of `EvmToken(A_MKR).identifier` and didn't notice what he was writting

Reply:
sure but still it's wrong, right? And you can fix it ?